webapp/src/components/sidebar_right/github_items.jsx

```diff
@@ -6,6 +6,7 @@ import PropTypes from 'prop-types';
 
 import {Badge, Tooltip, OverlayTrigger} from 'react-bootstrap';
 import {makeStyleFromTheme, changeOpacity} from 'mattermost-redux/utils/theme_utils';
+import Octicon, {Alert} from '@primer/octicons-react';
```

Comment:
@hanzei we should use our Mattermost icons.
`<i class='icon icon-alert-outline'/>`

Reply:
I've updated the icon. Does this look good to you?
![Screenshot from 2020-06-03 12-00-51](https://user-images.githubusercontent.com/16541325/83623768-07a0ac00-a592-11ea-8a6a-19ae031bafd5.png)

Reply:
@hanzei Looks a lot better, however, I feel like the spacing is slightly inconsistent on the right/left of the icon.
Can we make it equal (or visually equal)?

Reply:
Also, can we make the icon use our `errorTextColor` from the theme.

Reply:
@asaadmahmood I'm not sure about the color. It seams a bit to much for a warning. 1/5
![Screenshot from 2020-06-03 12-18-00](https://user-images.githubusercontent.com/16541325/83625526-53eceb80-a594-11ea-9d1f-90c91ddc7ec9.png)

Reply:
I think it looks okay. A conflict usually is a blocking action, and needs immediate attention if the PR is to be merged, so I'm okay it being the error text.
The design looks good to me now.

Reply:
Would you mind approving the PR?